spec/factories/file_attachment_file_revision_factory.rb

```diff
@@ -8,6 +8,7 @@
 
     transient do
       fixture { "text-file.txt" }
+      assets { nil }
```

Comment:
FileAttachment::FileRevision does not have many assets, it only `has_one` file_asset so this particular transient attribute is incorrect.